federation/source-config-handlers/src/main/java/org/codice/ddf/admin/sources/csw/CswSourceUtils.java

```diff
@@ -65,6 +66,15 @@
     private static final String GET_FIRST_OUTPUT_SCHEMA =
             "//ows:OperationsMetadata/ows:Operation[@name='GetRecords']/ows:Parameter[@name='OutputSchema' or @name='outputSchema']/ows:Value[1]/text()";
 
+    private RequestUtils requestUtils = new RequestUtils();
```

Comment:
Odd to instantiate here only to replace in one of your two ctors. Try this instead:
```java
private final RequestUtils requestUtils;

public CswSourceUtils() {
    this(new RequestUtils());
}

public CswSourceUtils(RequestUtils requestUtils) {
    this.requestUtils = requestUtils;
}
```